extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -94,7 +94,6 @@ export class DataTierApplicationWizard {
 
 			// don't open the wizard if connection dialog is cancelled
 			if (!this.connection) {
-				vscode.window.showErrorMessage(localize('dacfx.needConnection', 'Please connect to a server before using this wizard.'));
 				return;
```

Comment:
just to double check - this path wouldn't get hit in any case other than connection cancel - right?

Reply:
Yes this only gets hit if there is no connection after the connection dialog is closed